contrib/ledger-tests/instr.sh

```diff
@@ -31,8 +31,8 @@ build/native/gcc/bin/fd_ledger --cmd replay \
                             --allocator wksp \
                             --on-demand-block-ingest 1 \
                             --dump-insn-to-pb 1 \
```

Comment:
We should remove the instruction dumper and switch to using the transaction dumper